core/src/test/scala/kanaloa/reactive/dispatcher/Backends.scala

```diff
@@ -0,0 +1,18 @@
+package kanaloa.reactive.dispatcher
+
+import akka.actor.ActorRefFactory
+import akka.testkit.{TestActors, TestProbe}
+import kanaloa.reactive.dispatcher.queue._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait Backends {
```

Comment:
Way after the fact, but I wrote something a few PRs ago(which was briefly lived), which did what this was doing(deferring/controlling Routee creation).  We might want to switch back to using that, as while this is good for now, Thread.sleep will eventually get you into trouble.  Also, the other class allowed for the test to control exactly when a Backend was created, allowing for some granular testing.